basic/html/member/signup.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>회원가입 | 판다마켓</title>
+    <link rel="stylesheet" href="../../css/reset.css" />
+    <link rel="stylesheet" href="../../css/member.css" />
+</head>
+<body>
+    <div class="wrap">
+        <div class="member-area">
+            <div class="logo">
+                <h1>
+                    <span>판다마켓</span>
+                    <a href="../../">
```

Comment:
상대경로가 아닌 절대경로로 해주셔도 좋았을 것 같아요 
```suggestion
                    <a href="/">
```